src/index.d.ts

```diff
@@ -1,6 +1,6 @@
 declare module "local-devices" {
 
-    function findLocalDevices(address?: any): Promise<findLocalDevices.IDevice[]>;
+    function findLocalDevices(opts: { ipAddressOrRange?: any, skipNameResolution?: boolean, arpPath?: string }): Promise<findLocalDevices.IDevice[]>;
```

Comment:
I think this type needs to be updated to show that `opts` is optional (`opts?`).

Reply:
Very good spot!